modules/core/exported/client.go

```diff
@@ -92,6 +92,21 @@ type ClientState interface {
 		value []byte,
 	) error
 
+	// VerifyMultiHopMembership is a generic proof verification method which verifies a multi-hop proof of the existence of a value at a given CommitmentPath at the specified height.
+	// The caller is expected to construct the full CommitmentPath from a CommitmentPrefix and a standardized path (as defined in ICS 24).
+	VerifyMultihopMembership(
```

Comment:
I also had to add this client interface method. Possibly this would break things...